tests/vote_test.cpp

```diff
@@ -276,7 +276,7 @@ TEST_F(VoteTest, transfer_vote) {
   size_t step = 1000;
   auto vote = pbft_mgr2->generateVote(propose_block_hash, type, period, step);
 
-  nw2->sendPbftVote(nw1->getNodeId(), vote);
+  nw2->sendPbftVotes(nw1->getNodeId(), {vote});
```

Comment:
lets not use this architecture anymore. We already have a function `PacketsHandler::getSpecificHandler`. You could create the same function also inside `Network` and `TaraxaCapability` classes. Thanks to this we could delete most of the "just for tests" functions from network and TaraxaCapability and we would use new design instead:
`nw2->getSpecificHandler(SubprotocolPacketType::<type>).sendPbftVotes(nw1->getNodeId(), {vote});`

We can then create task to get rid of all of the "just for tests" methods

Reply:
fixed